adf/linkedService/AzureBlobStorage.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "AzureBlobStorage",
+	"type": "Microsoft.DataFactory/factories/linkedservices",
+	"properties": {
+		"annotations": [],
+		"type": "AzureBlobStorage",
+		"typeProperties": {
+			"connectionString": "DefaultEndpointsProtocol=https;AccountName=vdevoomopstorage;EndpointSuffix=core.windows.net;",
+			"encryptedCredential": "ew0KICAiVmVyc2lvbiI6ICIyMDE3LTExLTMwIiwNCiAgIlByb3RlY3Rpb25Nb2RlIjogIktleSIsDQogICJTZWNyZXRDb250ZW50VHlwZSI6ICJQbGFpbnRleHQiLA0KICAiQ3JlZGVudGlhbElkIjogIlYtREVWLU9NT1AtREZfY2FlZTdiMjItODQ0OC00YjlkLTk0NGQtNzUwNmYzZjUwMWJmIg0KfQ=="
```

Comment:
How are these json files being used? Also we should not check in credentials here. Should these files be in .gitignore?

Reply:
These json files were autogenerated by ADF as a config for connecting with various linked services. Good call out on the credentials, definitely need to look into how insecure is the encryptedCredentials and how to git track ADF pipelines without displaying encryptedCredentials as we work with ADF further